src/general/allowedrepos/constraint.tmpl

```diff
@@ -4,6 +4,7 @@ metadata:
   name: k8sallowedrepos
   annotations:
     metadata.gatekeeper.sh/title: "Allowed Repositories"
+    metadata.gatekeeper.sh/version: 1.0.0
```

Comment:
Can we add a validation to make sure this is bumped whenever the rego is updated?
Can we also add this to the developers doc to make sure devs bumps this when the rego is updated?

Reply:
I have added an [issue](https://github.com/open-policy-agent/gatekeeper-library/issues/241) to add a CI check. I'll open a PR another PR to address it.

Reply:
Also added a note about versioning in the README.